notebooks/SMAP/smap_issues_and_solutions.ipynb

```diff
@@ -0,0 +1,2724 @@
+{
```

Comment:
<p>I get the same error, this time for <code>root.visititems(get_vars)</code>:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">ValueError: Invalid location identifier (invalid identifier type to function)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
See comment above